doc/user_guide/eds.rst

```diff
@@ -580,6 +580,43 @@ background.
     Zr_Lb3
     background_order_6
 
+.. _eds_xray_databases-label:
+
+X-ray Line Databases
+^^^^^^^^^^^^^^^^^^^^^
+
+eXSpy supports two X-ray line energy databases to provide reference energies for X-ray lines:
+
+**XrayDB (default and recommended)**: An external database from the `XrayDB package <https://xraypy.github.io/XrayDB/>`_ 
+that provides high-precision X-ray line energies and related atomic data. XrayDB is maintained actively and 
+includes the most accurate and up-to-date values based on modern measurements and calculations.
+
+**Chantler2005 database**: A built-in database containing X-ray line energies derived from the 
+`NIST X-Ray Form Factor, Attenuation and Scattering Tables <https://www.nist.gov/pml/x-ray-form-factor-attenuation-and-scattering-tables>`_.
+
+The choice of database affects the reference energies used for X-ray line identification, peak fitting, 
+and energy calibration. XrayDB generally provides more accurate values and is therefore the default choice.
+
+You can specify which database to use when creating a model:
+
+.. code-block:: python
+
+    >>> # Use XrayDB (default, recommended)
+    >>> m = s.create_model(xray_line_source='xraydb')
+    
+    >>> # Use Chantler2005 database
+    >>> m = s.create_model(xray_line_source='Chantler2005')
+
+For example, the Fe Kα line energy differs between the databases:
+
+- **XrayDB**: 6.40521 keV (more precise, modern value)
+- **Chantler2005 database**: 6.4039 keV (legacy value)
+
+.. NOTE::
+   XrayDB is now a strict requirement for eXSpy and provides superior accuracy for quantitative 
```

Comment:
Silly question: how do we know that this is more accurate? Can we used a reference to support this statement?